### ISSUE TITLE
Create confusion matrix or matrices for multi-hot results

### ISSUE THREAD

**Issue by bencomp** [LeidenUniversityLibrary/MIDA-video-analysis#6]
We would like to see how often the models produce false positives and false negatives for each individual symbol, whereas TensorFlow's metrics only produce one overall accuracy/FP/FN scalar. Ideally we can get an indication whether symbols are confused.

If we consider each combination of symbols a class, creating a single confusion matrix should be easy.

**Comment by bencomp** [comment]
I wonder if it helped to create a model with multiple outputs (i.e. multiple single-node output layers) instead of a model with a single multi-node output layer. I just came across this example:

```
inputs = tf.keras.layers.Input(shape=(3,))
d = tf.keras.layers.Dense(2, name='out')
output_1 = d(inputs)
output_2 = d(inputs)
model = tf.keras.models.Model(
   inputs=inputs, outputs=[output_1, output_2])
model.compile(optimizer="Adam", loss="mse", metrics=["mae", "acc"])
model.fit(x, (y, y))
model.metrics_names
```
and it looks like that would allow us to track accuracy of predictions of each symbol separately. (Of course this example is too simple.)

Seeing the example I think this is what `"class_mode": "multi_output"` is meant for, in `ImageDataGenerator.flow_from_dataframe()`.
